lib/verbalize/action.rb

```diff
@@ -17,8 +17,19 @@ def self.included(target)
 
     module ClassMethods
       def input(*required_keywords, optional: [])
+        # treat `input` as an accessor if nothing is passed in.
+        return @inputs || [] if required_keywords.empty? && optional.empty?
+
+        @inputs = required_keywords
+        @optional_inputs = optional
+
         class_eval Build.call(required_keywords, Array(optional))
       end
+      alias_method :inputs, :input
```

Comment:
Please update the README to document the new functionality.

TODO: @taylorzr We should consider adding auto-generated yard or relish documentation to the project